test/inputs/input.reaction.ini

```diff
@@ -16,10 +16,10 @@ equation_system = 'navier-stokes'
 outdirBase = output
 
 [time]
-cfl = 0.5
+cfl = 0.2
 integrator = rk4
 enableConstantTimestep = True
-dt_fixed = 4.6e-4
+dt_fixed = 2.0e-4
```

Comment:
Did changing the quadrature lead to having to reduce `dt`?  Just wondering what happened here.

Reply:
It needed a smaller timestep, but did not need to change the reference solution. Didn't look into it much, as the rate is fictitious anyway.